test/functorch/test_control_flow.py

```diff
@@ -3545,6 +3545,144 @@ def cond_fn(x):
             self.assertEqual(cnt.frame_count, 3)
 
 
+_hop_schema_test_dtypes = [
+    "bool",
+    "int",
+    "float",
+    "str",
+    "tensor",
+    "symint",
+    "symbool",
+    "graph_module",
+    "script_obj",
+]
+
+
+class TestHopSchema(TestCase):
+    def _get_example_val(self, ty: str):
+        from torch.fx.experimental.sym_node import SymNode
+        from torch.fx.experimental.symbolic_shapes import ShapeEnv
+
+        def create_symtype(cls, pytype, shape_env, val):
+            from torch._dynamo.source import ConstantSource
+
+            symbol = shape_env.create_symbol(
+                val,
+                source=ConstantSource(
+                    f"__testing_hop_schema{len(shape_env.var_to_val)}"
+                ),
+                # dynamic_dim=DimDynamic.DYNAMIC,
+                # constraint_dim=None,
```

Comment:
what's going on?

Reply:
These shouldn't matter, will delete the commented part